packages/create-sitecore-jss/src/templates/nextjs-personalize/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "@sitecore-jss/sitecore-jss-tracking": "^20.0.0-canary.22",
+    "bootstrap": "^4.3.1",
+    "deep-equal": "^2.0.5",
+    "nprogress": "~0.2.0"
+  },
+  "devDependencies": {
+    "@sitecore-jss/sitecore-jss-dev-tools": "^20.0.0-canary.22",
+    "@sitecore-jss/sitecore-jss-manifest": "^20.0.0-canary.22",
+    "@types/deep-equal": "^1.0.1",
+    "@types/nprogress": "^0.2.0"
+  }
```

Comment:
Pretty sure the remainder are not necessary either (these might be leftover from old sample) in which case we could remove the package.json file entirely.